packages/browser/scripts/ci.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
-set -e
-
-echo '--- Build bundles'
-make build
-
-echo '--- Check Size'
-yarn run -T browser size-limit
```

Comment:
I wonder if we should include this into a CI step?

I guess this script used to be run on the old CI, but I never ported it when I originally ported stuff to our newer github actions.

Reply:
Yea I can add this to our CI action, should be as simple as running

```sh
npx size-limit
```

Reply:
It doesnt have to be this PR, but I do wonder if we'll lose the "mental note" to do this if we delete this unused ci.sh file.

Reply:
Added https://github.com/ht-sdks/events-sdk-js-mono/pull/9/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR38-R39